net/pfSense-pkg-pfBlockerNG/files/usr/local/www/pfblockerng/pfblockerng_alerts.php

```diff
@@ -427,7 +427,12 @@
 // Define common variables and arrays for report tables
 $fields_array	= $pfb_local = $pfb_localsub = $dnsbl_int = $local_hosts = array();
 
-$pfblines	= exec("/usr/local/sbin/clog {$filter_logfile} | {$pfb['grep']} -c ^");
+if (substr(trim(file_get_contents('/etc/version')), 0, 3) > '2.5') {
```

Comment:
Now that 21.02 and 2.5 are out, I don't think we'll be picking back any more updates to the 2.4.5 branch.
Also this check may fail as-is on Plus 21.02 anyhow.

Consider using another flag for testing this if you keep it, such as the presence of a file or function only found on 2.5.0. Also the `clog` binary will not be present on 2.5.0 systems.